src/rard/research/migrations/0070_updating_plain_renditions.py

```diff
@@ -0,0 +1,100 @@
+# Generated by Django 3.2 on 2024-03-08 12:27
+
+import re
+import string
+import unicodedata
+from django.db import migrations, models
+from django.utils.html import strip_tags
+
+
+def strip_combining_and_make_plain(content):
+    plain_text = make_plain_text(content)
+    normalized = unicodedata.normalize("NFD", plain_text)
+    return "".join(
+        [char for char in normalized if not unicodedata.combining(char)]
+    )
+
+
+def make_plain_text(content):
+    no_ufeff = content.replace("\ufeff", "")  # found around mentions for some reason
+    no_tags = strip_tags(no_ufeff.replace("><", "> <"))
+    no_html_chars = re.sub(r"&\w+;", " ", no_tags)
+    no_punctuation = no_html_chars.translate(str.maketrans("", "", string.punctuation))
+    no_lone_numbers = re.sub(r"\s\d{1,2}\s", " ", no_punctuation)  # mentions
+    no_excess_space = re.sub(r" +", " ", no_lone_numbers)
+    return no_excess_space
+
+
+def update_antiquarians(apps, schema_editor, plaining_function):
+    Antiquarian = apps.get_model("research", "Antiquarian")
+    for ant in Antiquarian.objects.all():
+        if ant.plain_introduction:
+            plaining_function(ant.introduction.content)
```

Comment:
Both plaining functions return a string which you're not doing anything with. You need to assign the result to `ant.plain_introduction`. Same for all the other models